app/src/main/java/org/eu/exodus_privacy/exodusprivacy/fragments/appdetail/AppDetailFragment.kt

```diff
@@ -132,11 +132,21 @@ class AppDetailFragment : Fragment(R.layout.fragment_app_detail) {
                         }
                     }
                 }
-                if (app.updated.isNotBlank()) {
-                    appReportTV.text = getString(
-                        R.string.report_date,
-                        viewModel.getFormattedReportDate(app.updated, view.context)
-                    )
+                if (app.created.isNotBlank()) {
+                    if (viewModel.getFormattedReportDate(app.created, view.context) != viewModel.getFormattedReportDate(app.updated, view.context)) {
```

Comment:
`viewModel.getFormattedReportDate(app.created, view.context)` & `viewModel.getFormattedReportDate(app.updated, view.context)` should be aggregated in variables to avoid the multiple call to the `getFormattedReportDate` method.

Reply:
Fixed https://github.com/Exodus-Privacy/exodus-android-app/commit/bab6368dceac274e18d8736042b574b21bc732d4